konami.js

```diff
@@ -0,0 +1,41 @@
+  var konamiCodePosition = 0;
```

Comment:
First of all Kyle, you are crazy..I mean Krazy...with a capital K.  AND, this is awesome.  Nice job slicing off a piece and serving it up hot.